galaxy_wrappers/CollectResults/glob_res.pl

```diff
@@ -101,6 +101,8 @@
     map { $exist_part_used{ $_->[0] } = 1 } @{$used_cm};
 }
 
+system("cp $combined_cm combined_cm_out");
```

Comment:
If there is no $combined_cm argument from the input then this copy fails, you should copy only if $combined_cm != combined_cm_out. do you agree?